### PR TITLE
[MIG] Translated changes from 7.0 branch

### PR DESCRIPTION
**Pull request by pedrobaeza**
After the PEP8 fix. Also a new module for 7.0 lands in `__unported__` dir.

### CODE REVIEW

**Comment by hbrunn** [comment]
ah, I think given the modules live in **unported**, it doesn't hurt, but could you set installable: False for good order?

**Comment by pedrobaeza** [comment]
You're right. Done.

**Comment by hbrunn** [comment]
:+1:

File: __unported__/document_page_multi_company/__openerp__.py [hunk 1]
```diff
@@ -40,3 +40,4 @@
     'auto_install': False,
     'images': [],
 }
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (bwrsandman) [inline __unported__/document_page_multi_company/__openerp__.py]: Don't need this addition

Comment (pedrobaeza) [inline __unported__/document_page_multi_company/__openerp__.py]: This is because I have copied as is from 7.0. In 7.0 must be this line. I'm removing it.

**Comment by pedrobaeza** [comment]
I have removed comments for all files.

**Comment by gurneyalex** [comment]
:+1:

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**